src/plugins/irc-events/mode.js

```diff
@@ -39,6 +39,17 @@ module.exports = function (irc, network) {
 		targetChan.pushMessage(client, msg);
 	});
 
+	irc.on("user info", function (data) {
+		const serverChan = network.channels[0];
```

Comment:
do we want to show this in the open channel if it exists?

Reply:
If you prefer?
I wasn't quite sure what to do with it, so I shoved it into the server buffer

Reply:
Yeah, I think this should go in `activeChannel`, similar to notices targeted at the user.